scripts/postman_test_automation.sh

```diff
@@ -19,16 +19,23 @@ get_api_keys() {
     # [<connector_name>]
     # api_key = "HeadKey of <connector_name>"
 
-    API_KEY=$(echo "${result}" | awk -F ' = ' '$1 == "api_key" { print $2 }')
-    KEY1=$(echo "${result}" | awk -F ' = ' '$1 == "key1" { print $2 }')
-    API_SECRET=$(echo "${result}" | awk -F ' = ' '$1 == "api_secret" { print $2 }')
-
+    # Keys are exported as environment variables since some API Keys for connectors such as ACI
+    # base Base64 based and requires "Bearer" to be prefexed such as "Bearer Skst45645gey5r#&$==".
```

Comment:
Is there an error around this part?

> [...] such as ACI base Base64 based and requires "Bearer" to be prefexed [...]

Also, there's a typo:

```suggestion
    # Keys are exported as environment variables since some API Keys for connectors such as ACI
    # base Base64 based and requires "Bearer" to be prefixed such as "Bearer Skst45645gey5r#&$==".
```

Reply:
Auto fill issue.